### DISCUSSION TITLE
Bouncing size when the toolbar changes

### DISCUSSION THREAD

**Discussion by devopg** [Milkdown/milkdown#676] [Ideas]
Bouncing size when the toolbar changes
If you place milkdown in the center margin: 0 auto; when the set of icons in the toolbar changes make bouncing.

https://user-images.githubusercontent.com/61819835/184550199-ea64ee7f-ff58-47ec-947e-eeda9af932a7.mp4

if set width for .milkdown looks terrible

![image](https://user-images.githubusercontent.com/61819835/184550193-a670ad62-488a-4611-94f2-4aeaecb7b17e.png)

**Comment by Saul-Mirone** [comment]
You can fix the width of `.milkdown-menu-wrapper`.

**Reply by devopg** [reply]
when will 
![image](https://user-images.githubusercontent.com/61819835/184587247-46d55d50-d33f-4d73-ac79-09062f243cc1.png)

I think better made that toolbar cant change main editor like. Take all editor width with overflow-x-scroll

**Reply by Saul-Mirone** [reply]
The `div[.milkdown-menu-wrapper]` wraps both the editor and the menu. Not only the menu.
